client/my-sites/importer/section-import.jsx

```diff
@@ -271,10 +271,11 @@ class SectionImport extends Component {
 		const { jetpack: isJetpack } = site;
 		const headerText = translate( 'Import your content' );
 		const subHeaderText = translate(
-			'Bring content hosted elsewhere over to WordPress.com. ' +
+			'Bring content hosted elsewhere over to WordPress.com.{{br/}}' +
```

Comment:
It would be great if we could do this without changing the translatable string. Why not use a `display:block` for the link?